stable/artifactory-ha/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 All changes to this chart will be documented in this file.
 
 ## [0.13.5] - Jun 5, 2019
+* Updated Google Cloud Storage API URL and https settings
```

Comment:
Need a dedicated new version for your change.

Reply:
Done. You can check if it looks good